test-d/classic.tsx

```diff
@@ -40,7 +39,8 @@ expectError(<a invalid={{}} />)
 expectError(<a invalid={[1]} />)
 expectError(<a>{{invalid: 'child'}}</a>)
 
+// This is where the classic runtime differs from the automatic runtime.
```

Comment:
Can you expand on these comments (in classic and automatic), what exactly that difference is?